lib/parser.js

```diff
@@ -60,6 +60,7 @@ class Parser extends EventEmitter {
       entry.value = entry.value[entry.name]
 
       if (property.key.name === 'computed') {
+        /* istanbul ignore else */
```

Comment:
Hum... I don't like `istanbul ignore ...` statement. It may a problem if we add the `else` statement and forget to remove the `istanbul` statement. I prefer to add a unit test to cover the `else` branch